libs/src/api/Track.ts

```diff
@@ -380,6 +380,104 @@ export class Track extends Base {
 
```

Comment:
I'll clean up this file as a followup, but I want to get this merged to have us all able to query the right endpoints first